lib/simulator-xcode-9.js

```diff
@@ -164,6 +166,20 @@ class SimulatorXcode9 extends SimulatorXcode8 {
     }
   }
 
+  /**
+   * Configure keyboard preference as 'com.apple.keyboard.preferences' domain via 'defaults' command.
+   */
+  async configureKeyboardPreference () {
+    const argChunks = generateDefaultsCommandArgs({
+    // To disable 'DidShowContinuousPathIntroduction' for iOS 15+ simulators since changing the preference via WDA
+    // does not work on them. Lower than the versions also can have this preference, but nothing happen.
+      DidShowContinuousPathIntroduction: 1
+    }, true);
+    await B.all(argChunks.map((args) => this.simctl.spawnProcess([
+      'defaults', 'write', 'com.apple.keyboard.preferences', ...args
```

Comment:
i know we normally try to put magic strings like `com.apple.keyboard.preferences` as constants?